model_analyzer/analyzer.py

```diff
@@ -299,20 +320,26 @@ def _profile(self, perf_analyzer, dcgm_monitor):
             perf_records = []
             time.sleep(self._duration_seconds)
         dcgm_records = dcgm_monitor.stop_recording_metrics()
+        cpu_records = cpu_monitor.stop_recording_metrics()
 
         # Insert all records into aggregator and get aggregated DCGM records
         record_aggregator = RecordAggregator()
         for record in dcgm_records:
             record_aggregator.insert(record)
 
+        cpu_record_aggregator = RecordAggregator()
```

Comment:
Why can't we use the same Record Aggregator here?

Reply:
Wanted to make explicit the where the cpu records are coming from in `Analyzer.profile_model` but I agree it can be combined with `perf_record_aggregator`. I will fix that.

Reply:
how/where is cpu_record_aggregator consumed/used?

Reply:
It was previously aggregated and returned from the function.